app/workers/asset_manager_create_asset_worker.rb

```diff
@@ -3,15 +3,15 @@ class AssetManagerCreateAssetWorker < WorkerBase
 
   sidekiq_options queue: "asset_manager"
 
-  def perform(temporary_location, asset_params, draft = false, attachable_model_class = nil, attachable_model_id = nil, auth_bypass_ids = [])
+  def perform(temporary_location, asset_params, draft = false, attachable_model_class = nil, attachable_model_id = nil, auth_bypass_ids = [], access_limited_organisation_ids = [])
     return unless File.exist?(temporary_location)
 
     file = File.open(temporary_location)
 
     assetable_id, assetable_type, asset_variant = asset_params.values_at("assetable_id", "assetable_type", "asset_variant")
-    asset_options = { file:, auth_bypass_ids:, draft: }
-    authorised_user_uids = get_authorised_user_ids(attachable_model_class, attachable_model_id)
-    asset_options[:access_limited] = authorised_user_uids if authorised_user_uids
+    asset_options = { file:, auth_bypass_ids:, draft:, access_limited_organisation_ids: }
+    authorised_organisation_uids = get_authorised_organisation_ids(attachable_model_class, attachable_model_id)
+    asset_options[:access_limited_organisation_ids] = authorised_organisation_uids if authorised_organisation_uids
```

Comment:
I'm slightly confused by this because we set `asset_options[:access_limited_organisation_ids]` on line 12, then overwrite it on line 14?